nrf52811-hal/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "nrf52811-hal"
+version = "0.11.1"
+edition = "2018"
+description = "HAL for nRF52811 microcontrollers"
+repository = "https://github.com/nrf-rs/nrf-hal"
+authors = [
+    "James Munns <james@onevariable.com>",
+    "Hanno Braun <hanno@braun-robotics.com>",
+    "John Scarrott <johnps@outlook.com>",
+    "Wez Furlong <wez@wezfurlong.org>",
+    "Ferdia McKeogh <ferdia@mckeogh.tech>",
+]
+categories = ["embedded", "hardware-support", "no-std"]
+keywords = ["arm", "cortex-m", "nrf52", "hal", "nrf52811"]
+license = "MIT OR Apache-2.0"
+
+[dependencies]
+nrf52811-pac = "0.9.1"
+
+[dependencies.nrf-hal-common]
+path = "../nrf-hal-common"
+default-features = false
+features = ["52811"]
+version = "=0.11.1"
+
+[dependencies.embedded-hal]
+features = ["unproven"]
+version = "0.2.3"
+
+[features]
+doc = []
```

Comment:
This seems unused

Reply:
Do you mean the embedded-hal? Not sure whether GH is broken or I missunderstand because of other reasons ;)

Reply:
I meant that the `doc` feature seems unused

Reply:
Ah, I see :) I guess it's unused in the 810 crate then too :)